src/components/ChatRoomPage/Chat.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IChat } from '../../interface/interface';
+import { useRecoilValue } from 'recoil';
+import { currentUserState } from '../../state/atom';
+import userData from '../../db/userData.json';
+
+const Chat = (chat: IChat) => {
+  const currentUser = useRecoilValue(currentUserState);
+  //chat.userId === currentUser.userId;
+
+  return (
+    <ChatContainer isActive={chat.userId === currentUser.userId}>
+      <UserImg
+        src={`${process.env.PUBLIC_URL}/Imgs/${
+          userData[chat.userId].userImg
+        }.jpg`}
```

Comment:
```suggestion
        src={`Imgs/${currentUser.userImg}.jpg`}
```

으로 수정하면 좀 더 간단하게 이미지를 불러올 수 있습니다!!